lib/matplotlib/widgets.py

```diff
@@ -3245,7 +3249,8 @@ class RectangleSelector(_SelectorWidget):
     See also: :doc:`/gallery/widgets/rectangle_selector`
     """
 
-    def __init__(self, ax, onselect, *, minspanx=0, minspany=0, useblit=False,
+    def __init__(self, ax, *, onselect=None, minspanx=0,
```

Comment:
```suggestion
    def __init__(self, ax, onselect=None, *, minspanx=0,
```
The `*` makes it keyword only which would require deprecation etc. I think it make sense to provide a default value, but not require old code to pass `onselect` with a keyword. (I guess you just forgot about this line.)